beacon/blockchain/execution_engine.go

```diff
@@ -30,10 +30,7 @@ import (
 )
 
 // sendPostBlockFCU sends a forkchoice update to the execution client after a
-// block is finalized.This function should only be used to notify
-// the EL client of the new head and should not request optimistic builds, as:
-// Optimistic clients already request builds in handleOptimisticPayloadBuild()
-// Non-optimistic clients should never request optimistic builds.
```

Comment:
IMO this comment is not clear nor really informative. Please correct me if I am wrong/ you disagree